### DISCUSSION TITLE
Can we use Waveshare ESP32-C3 Zero to Run Pixelit

### DISCUSSION THREAD

**Discussion by Techposts** [pixelit-project/PixelIt#357] [General]
I have this board with me can I use this for Pixelit? 
https://www.waveshare.com/esp32-c3-zero.htm
is this going to work well or better than D1 Mini (ESP8266) that I am currently using? 

Please help!

**Comment by foorschtbar** [comment]
The code on ESP32 and ESP8266 is the same. No improvements with ESP32.

**Comment by Techposts** [comment]
But it’s not running on ESP32C3 Supermini. That’s the issue. Also, the Pixelit in D1 Mini stops sending mqtt messages after sometime and keeps displaying only time and date. It does not even display the Temp and Humidity from the installed DHT sensor until I restart it.

**Reply by foorschtbar** [reply]
Had you some serial or web log output for us?

**Comment by Techposts** [comment]
I tried all ESP32 firmware, but none works on ESP 32 C3 Supermini. No log output after a successful flash, the chip keeps rebooting. 

![image](https://github.com/user-attachments/assets/38bd1c72-d2be-4896-89dd-aeb0b2988b3a)

**Reply by foorschtbar** [reply]
We had no firmware for ESP-Cx, its a different architecture than the normal ESP32.
